spikeinterface/sorters/runsorter.py

```diff
@@ -139,21 +139,40 @@ def find_recording_folder(d):
 
 class ContainerClient:
     """
-    Small absctarction to run commands in:
+    Small abstraction to run commands in:
       * docker with "docker" python package
       * singularity with  "spython" python package
     """
     def __init__(self, mode, container_image, volumes, extra_kwargs):
         assert mode in ('docker', 'singularity')
         self.mode = mode
+
         if mode =='docker':
             import docker
             client = docker.from_env()
+            if extra_kwargs.get('requires_gpu', False):
+                extra_kwargs.pop('requires_gpu')
+                extra_kwargs["device_requests"] = [
+                    docker.types.DeviceRequest(count=-1, capabilities=[['gpu']])]
```

Comment:
I don't see any equivalent to this for singularity. @samuelgarcia Do you have any experience with using singularity with gpus?